src/app/shared/components/xterm-terminal/terminal-status-bar/theme.scss

```diff
@@ -0,0 +1,32 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@use 'sass:map';
+
+@mixin theme-terminal-status-bar-component($colors) {
+  .toolbar-status-container {
+    background-color: map.get($colors, terminal-status-bar-background);
+    border-bottom: 1px solid map.get($colors, terminal-status-bar-border-color);
+    color: map.get($colors, terminal-status-bar-color);
+
+    // 'X' icon button
+    & .close-terminal-button:hover {
+      background-color: map.get($colors, terminal-status-bar-close-button-hover) !important;
```

Comment:
In most of the cases `!important` can be replaced with usage of specific class selectors. Also, the comment above the rule is unnecessary.